docs/docs.polserver.com/pol100/objref.xml

```diff
@@ -54,8 +54,8 @@
 <method proto="getprop(string propname)" returns="script object/error">Returns an unpacked script object (i.e. int,string,array,etc)</method>
 <method proto="setprop(string propname, object propval)" returns="true/error">Sets a packable object to a property.</method>
 <method proto="eraseprop(string propname)" returns="true/error">Erases the property named 'propname'.</method>
-<method>Sets the built-in member 'membername' to 'value'. obj.set_member(&quot;name&quot;,&quot;Eric&quot;) is the same as obj.name := &quot;Eric&quot;" proto="set_member(string membername, obj value)" returns="object, new value of member or error</method>
-<method>Gets the value of the built-in member 'membername'. var objname := obj.get_member(&quot;name&quot;) is the same as var objname := obj.name" proto="get_member(string membername)" returns="object, value of member or error</method>
+<method proto="set_member(string membername, obj value)" returns="object, new value of member or error">Sets the built-in member 'membername' to 'value'. obj.set_member(&quot;name&quot;,&quot;Eric&quot;) is the same as obj.name := &quot;Eric&quot;" proto="set_member(string membername, obj value)" returns="object, new value of member or error</method>
+<method proto="get_member(string membername)" returns="object, value of member or error">Gets the value of the built-in member 'membername'. var objname := obj.get_member(&quot;name&quot;) is the same as var objname := obj.name" proto="get_member(string membername)" returns="object, value of member or error</method>
```

Comment:
Seems my parser broke it :)
But the method text still contains the proto/returns attributes